### ISSUE TITLE
customizer won't accept my iso 

### ISSUE THREAD

**Issue by zakariaGatter** [kamilion/customizer#117]
i want work wit Customizer on ubuntu server 14.06 64bit but when i select the iso Customizer wont accept it 
there is any method to work with ubuntu 14.04 on Customizer ???

**Comment by ghost** [comment]
@zakariaGatter 
Hi, we did mention in our Wiki page already. You probably didn't read the [First guide](https://github.com/clearkimura/Customizer/wiki/First-guide) or [FAQ](https://github.com/clearkimura/Customizer/wiki/FAQ) page.

Have you at least read the FAQ page?

> **2. What are compatible distros and releases?**
> 
> Any Ubuntu-based, Live CD ISO images are [supported images](https://github.com/clearkimura/Customizer/wiki/Supported-images). The smallest supported image is Ubuntu Mini Remix. For known issues in recent releases of Ubuntu ISO image, see issue #76.

If you did, you will likely notice that the "supported images" page covered this already.

> **ISO image compatibility**
> 
> [...] Other ISO images such as alternate, server and minimal are [different and unsupported](https://github.com/clearkimura/Customizer/wiki/Different-images).

**TL;DR** Server ISO images are unsupported.
